Microsoft.Toolkit.Uwp.Notifications/Microsoft.Toolkit.Uwp.Notifications.csproj

```diff
@@ -3,7 +3,7 @@
   <PropertyGroup>
     <!--<TargetFrameworks>netstandard1.4;uap10.0;native;net461;netcoreapp3.1</TargetFrameworks>-->
     <!-- Removed 'native' target to unblock CI on VS 16.8, tied to changes breaking workaround for https://github.com/NuGet/Home/issues/5154 -->
-    <TargetFrameworks>netstandard1.4;uap10.0;net461;netcoreapp3.1</TargetFrameworks>
+    <TargetFrameworks>netstandard1.4;uap10.0.19041;net461;netcoreapp3.1;net5.0;net5.0-windows10.0.17763.0</TargetFrameworks>
```

Comment:
@clairernovotny when we specify `uap10.0.19041` here will to be enforced for the "Target" or the "Min" required version of the app?

In our other packages we assume it's the min and have `17763`, so I would assume we want to follow that pattern here as well?

@andrewleader was there some guidance or doc you reference in setting this up?

Reply:
`uap10.0.19041` is the target, not the min. Min version is specified in the `<TargetPlatformMinVersion>10.0.10240.0</TargetPlatformMinVersion>`, I've tested and confirmed it installs on old UWPs. I don't think the MSBuild.Sdk.Extras has documented all of this, but it works 🤷‍♂️

That's how the net5.0-windows TF works too, what's specified is the target... however, there's a weird case where if the app is targeting `17763` and the library is targeting `19041` but ALSO has a plain `net5.0` target, it'll pick `net5.0` instead... hence why I've kept `net5.0-windows` at 17763 (I confirmed you do not need the 17763 SDK installed to build it though so ppl only need 19041 SDK).